### ISSUE TITLE
Usage could be explained a bit more

### ISSUE THREAD

**Issue by Mindavi** [siraben/nix-lint#1]
I tried (naively) to install npm using `nix-shell -p npm` and then doing `npm run lint <path>`, but that didn't work.

Now I tried next to do `npm install`, but that failed because it expected python3 in the PATH.

After going into a shell with `nix-shell -p nodejs python3`, running `npm install` and then running `npm run lint <path>`, it worked.

Adding a `shell.nix` might go a long way :), but just expanding the instructions should be fine too.

# TLDR

Usage:

- `nix-shell -p nodejs python3`
- `npm install`
- `npm run lint <path>`

Thanks for working on this! Expanding it seems very easy, just added a rule for makeWrapper which causes an eval failure when it's in buildInputs.

**Comment by siraben** [comment]
Thanks for the feedback! I'll figure out a way to write a shell.nix or make this a flake.
